hiro.proto

```diff
@@ -2257,3 +2266,87 @@ message SyncResponse {
   // Current server time.
   int64 current_time_sec = 11;
 }
+
+// Represents an available reward based on the progress of a streak.
+message StreakAvailableReward {
+  // Minimum streak progress count required for this reward, inclusive.
+  int64 count_min = 1;
+  // Maximum streak progress count required for this reward, inclusive.
+  int64 count_max = 2;
+  // Available reward contents.
+  AvailableRewards reward = 3;
+}
+
+// A single claimed streak reward.
+message StreakReward {
+  // Minimum streak progress count required for this reward, inclusive.
+  int64 count_min = 1;
+  // Maximum streak progress count required for this reward, inclusive.
+  int64 count_max = 2;
+  // Reward that was actually granted.
+  Reward reward = 3;
+  // The UNIX time (for gRPC clients) or ISO string (for REST clients) when this reward was claimed.
+  int64 claim_time_sec = 4;
+}
+
+// An individual streak, along with its status and progress if any.
+message Streak {
+  // Unique identifier.
+  string id = 1;
+  // Display name for this streak.
+  string name = 2;
```

Comment:
is it possible to also get a description as well as a name?

Reply:
Will add. 👍

Reply:
Thank you